fmf/base.py

```diff
@@ -794,3 +822,18 @@ def __getitem__(self, key):
             return self.children[key[1:]]
         else:
             return self.data[key]
+
+    def __len__(self) -> int:
+        return len(self.children) + len(self.data)
+
+    def __iter__(self) -> Iterator[Any]:
+        for c in self.children:
+            yield f"/{c}"
+        for d in self.data:
+            yield d
+
+    def __contains__(self, item: str):
+        if item.startswith("/"):
+            return item[1:] in self.children
+        else:
+            return item in self.data
```

Comment:
These interfaces I am not sure about. Should they be combined here, or just output the `children` data. In particular `len()` feels dubious.

Reply:
Also missing interfaces: `keys`, `values`, `items`, `has_key` and probably equivalents to distinguish between data and children